### PR TITLE
New Twitter Cheat Sheet

### PR DESCRIPTION
**Pull request by luthfii**
This is a New Instant Answer:
New Twitter Cheat Sheet

Description of new Instant Answer:
Twitter Keyboard Shortcuts

--------------
Instant Answer Page: https://duck.co/ia/view/twitter_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/twitter.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+    "id": "twitter_cheat_sheet",
+    "name": "Twitter Cheat Sheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/twitter.json]: "Twitter" would be enough here

Comment (luthfii) [inline share/goodie/cheat_sheets/json/twitter.json]: For both id and name? Should i change directly here? or create new pull request?

Comment (jophab) [inline share/goodie/cheat_sheets/json/twitter.json]: No. I mean for the name, "Twitter" would be enough . You can edit in this branch and give a PR

File: share/goodie/cheat_sheets/json/twitter.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
+{
+    "id": "twitter_cheat_sheet",
+    "name": "Twitter",
+    "description": "Twitter Keyboard Shortcuts",
+    "metadata": {
+        "sourceName": "Twitter",
+        "sourceUrl": "https://twitter.com"
+    },
+    "aliases": [
```
Comment (ankushg07) [inline share/goodie/cheat_sheets/json/twitter.json]: @luthfii  @jophab Do we need these alias for twitter?? Are they useful?

File: share/goodie/cheat_sheets/json/twitter.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            "val": "Lists",
+            "key": "[g] [i]"
+        }, {
+            "val": "Messages",
+            "key": "[g] [m]"
+        }, {
+            "val": "Settings",
+            "key": "[g] [s]"
+        }, {
+            "val": "Go to user ...",
```
Comment (ankushg07) [inline share/goodie/cheat_sheets/json/twitter.json]: @luthfii @jophab I think [g][s] goes to settings only "Go to user ..." is not valid

Comment (luthfii) [inline share/goodie/cheat_sheets/json/twitter.json]: ok, thanks for the corrections

File: share/goodie/cheat_sheets/json/twitter.json (hunk 2 above)
Comment (jophab) [inline share/goodie/cheat_sheets/json/twitter.json]: I was not sure about it that day. 
Its better to remove the alias part.
I see now its has been removed. Good catch @ankushg07 :+1: 
@luthfii Thanks for making quick corrections :+1:

File: share/goodie/cheat_sheets/json/twitter.json [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+            "val": "Reply",
+            "key": "r"
+        }, {
+            "val": "Retweet",
+            "key": "t"
+        }, {
+            "val": "Direct message",
+            "key": "m"
+        }, {
+            "val": "Mute User",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/twitter.json]: lets change to "Mute user"
Do the same for the following val fields also. That is we can avoid mixed use of uppercase and lowercase letters in val fields

File: share/goodie/cheat_sheets/json/twitter.json [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    	   "val": "Previous Tweet",
+    	   "key": "k"
+    	}, {
+           "val": "Page down",
+           "key": "Space"
+    	}, {
+           "val": "Load new Tweets",
+    	   "key": "."
+    	}],
+        "Timelines": [{
```
Comment (ankushg07) [inline share/goodie/cheat_sheets/json/twitter.json]: "Timeline " would be more appropriate

**Comment by pjhampton** [comment]
This looks like it has gone stale. Closing as a result. 

We are trying to direct the community's attention towards more substantial and impactful issues that will benefit many users 🚀  check out the [following issues table here](https://duckduckhack.com/issues)! 😄

**Pull request closed without merging**